saos-persistence/src/main/java/pl/edu/icm/saos/persistence/model/importer/notapi/RawSourceAcJudgment.java

```diff
@@ -0,0 +1,15 @@
+package pl.edu.icm.saos.persistence.model.importer.notapi;
+
+import javax.persistence.Cacheable;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+/**
+ * @author madryk
+ */
+@Table(schema="importer", name="notapi_raw_source_nac_judgment")
```

Comment:
I think we'd rather use 'notapi_raw_source_ac_judgment' so the name would be compatible.